src/sources/kubernetes_logs/mod.rs

```diff
@@ -49,13 +50,15 @@ use crate::{
 mod k8s_paths_provider;
 mod lifecycle;
 mod namespace_metadata_annotator;
+mod node_metadata_annotator;
 mod parser;
 mod partial_events_merger;
 mod path_helpers;
 mod pod_metadata_annotator;
 mod transform_utils;
 mod util;
 
+use crate::sources::kubernetes_logs::node_metadata_annotator::NodeMetadataAnnotator;
```

Comment:
```suggestion
use self::node_metadata_annotator::NodeMetadataAnnotator;
```

It would be cleaner to update the `use` like this, which is better than what we did for the other `use`s below (as it also helps to inform that these aren't external). If you wanted to fixup the others I wouldn't mind 😄